src/streamlit/pages/settings.py

```
@@ -0,0 +1,19 @@
+import streamlit as st
+
+def settings_page():
+    st.title("Settings")
+    
+    st.header("User Preferences")
+    st.subheader("Notification Settings")
+    email_notifications = st.checkbox("Enable Email Notifications", value=True)
```

Ruff (0.12.2)

Local variable `email_notifications` is assigned to but never used

Remove assignment to unused variable `email_notifications`

(F841)